bower.json

```diff
@@ -34,5 +35,8 @@
         "sinon": "http://sinonjs.org/releases/sinon-1.17.0.js",
         "angular-mocks": "1.4.4"
     },
-    "private": true
+    "private": true,
+    "resolutions": {
```

Comment:
adding a note to address a similar issue in https://github.com/dimagi/commcare-hq/pull/9231